internal/pkg/controllers/profile/profile.go

```diff
@@ -90,13 +92,18 @@ func Setup(mgr ctrl.Manager, l logr.Logger) error {
 
 // A Reconciler reconciles seccomp profiles.
 type Reconciler struct {
-	client client.Client
-	log    logr.Logger
-	record event.Recorder
+	client          client.Client
+	log             logr.Logger
+	record          event.Recorder
+	stopReconciling bool
 }
 
 // Reconcile reconciles a ConfigMap representing a seccomp profile.
 func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error) {
+	if r.stopReconciling {
+		return reconcile.Result{}, nil
+	}
```

Comment:
@hasheddan do you think it makes sense to generally stop reconciling like this? The only way to restart the reconciling would be to restart the workload now.

Reply:
I don't think the benefit of skipping running the checks on the node on very infrequent reconciles is worth the added complexity of managing this / having to restart the workload

Reply:
Sounds good, let me revert that change.